docs/verify_cri.md

```diff
@@ -0,0 +1,100 @@
+# Verify pouch CRI by the cri-tools
+
+[cri-tools](https://github.com/kubernetes-incubator/cri-tools) provides a CLI([crictl](https://github.com/kubernetes-incubator/cri-tools/blob/master/docs/crictl.md)) for CRI-compatible container runtimes. This is an easy way to verify pouch CRI without needing to set up Kubernetes components.
```

Comment:
I think we had better use `pouch CRI implementation` rather than `pouch CRI`. As a result, the sentence could be `This is an easy way to verify CRI implementation in pouch without setting up all Kubernetes components.`